### PR TITLE
use geofence attribute to permit per polyline distance configuration

### PR DESCRIPTION
**Pull request by ivanfmartinez**

### CODE REVIEW

File: src/org/traccar/model/Geofence.java [hunk 1]
```diff
@@ -65,7 +65,9 @@ public void setArea(String area) throws ParseException {
         } else if (area.startsWith("POLYGON")) {
             geometry = new GeofencePolygon(area);
         } else if (area.startsWith("LINESTRING")) {
-            geometry = new GeofencePolyline(area, Context.getConfig().getDouble("geofence.polylineDistance", 25));
+            final double distance = getDouble("polylineDistance");
+            geometry = new GeofencePolyline(area, distance >= 1 ? distance
```
Comment (tananaev) [inline src/org/traccar/model/Geofence.java]: Why `>= 1`?

**Comment by ivanfmartinez** [comment]
it can be just > 0, but as I had before some precision errors during floating point comparision I considered this as a good value without precision errors, and non one will use 0.05 as precision for geofence....

What value do you want to be there ?

**Comment by tananaev** [comment]
There is no precision error with zero value, which is probably what you want to check to make sure it's set.

**Comment by tananaev** [comment]
Merged, thanks.

**Pull request merged**